sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -148,6 +193,15 @@ private[parquet] object ParquetFilters {
     case BinaryType =>
       (n: String, v: Any) =>
         FilterApi.gtEq(binaryColumn(n), Binary.fromReusedByteArray(v.asInstanceOf[Array[Byte]]))
+    case DateType =>
```

Comment:
Add a new SQLConf to make it configurable. Users can turn it off if they hit a bug introduced in this PR

Reply:
Have added, kindly help review.